### PR TITLE
added my name to README.md

### PR DESCRIPTION
**Pull request by marva-mehr**

### CODE REVIEW

**Review by marvaaaa — COMMENTED** [review]
add new files

File: home.index.html
Comment (guadalupe-alican) [inline home.index.html:1]: This file should be named index.html not home.index.html

File: pay.index.html
Comment (guadalupe-alican) [inline pay.index.html:1]: This should not be called pay.index.html you can name it just pay.html

File: store.index.html
Comment (guadalupe-alican) [inline store.index.html:1]: This should be called store.html

File: store.styles.css
Comment (guadalupe-alican) [inline store.styles.css:1]: This should be called store.css

**Review by ElnazMohmand — APPROVED** [review]

**Pull request merged**